scripts/beaker/unshard.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+IFS=$'\n\t'
+
+# Setup Python environment.
+conda shell.bash activate base
+
+# Install flash-attn
+pip install awscli
+pip install '.[train]'
+pip freeze
+
+# Move AWS credentials from env to relevant files
+mkdir -p ~/.aws
+printenv AWS_CONFIG > ~/.aws/config
+printenv AWS_CREDENTIALS > ~/.aws/credentials
+
+checkpoint=ai2-llm/checkpoints/OLMo-medium/amberish7/step477850
```

Comment:
This wasn't the purpose of your PR, but by making this a parameter you can make this script easily usable by everyone else

Reply:
Done: ba67ac0e